services/wireguard/endpoint/netstack-provider/private.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package netstack_provider
+
+import (
+	"net"
+	"strings"
+
+	"github.com/mysteriumnetwork/node/config"
+	"gvisor.dev/gvisor/pkg/tcpip"
+)
+
+func mustParseCIDR(cidrs []string) []*net.IPNet {
+	ipnets := make([]*net.IPNet, len(cidrs))
+	for i, cidr := range cidrs {
+		_, ipnet, err := net.ParseCIDR(cidr)
+		if err != nil {
+			panic(err)
+		}
+		ipnets[i] = ipnet
+	}
+	return ipnets
+}
+
+var privateIPv4Block []*net.IPNet
```

Comment:
This is a global variable, there might be a race condition when several consumers connect at the same time, this will be concurrently overwritten. Don't use a global variable, you can just get it from the config for every session creation.